content/blog/_index.md

```diff
@@ -1,68 +1,105 @@
 ---
 
-title: "Blog"
-description: "Blog posts"
-draft: false
+title: "Blog" description: "Blog posts" draft: false
```

Comment:
@Water261 revert this change, it's required frontmatter.

Reply:
(actually just revert everything in the file, I'm guessing your editor got excited?)